docs/reference/TensorFlow.md

```diff
@@ -1,405 +1,394 @@
+    1. Save the file.
 
--   Modify the code and deploy to the device.
+1. Copy the `tiny_conv_micro_features_model_data.cc` <!--where is this file? it's not the one I was just editing-->file into the `tensorflow/tensorflow/lite/micro/tools/make/gen/mbed_cortex-m4/prj/micro_speech/mbed/tensorflow/lite/micro/examples/micro_speech/micro_features` directory.
 
+## Modify the application code
 
-Note: Building TensorFlow and training the model will each take a couple of hours on an average computer. We will not perform this at this stage. For a full guide on how to do this, refer to the [Supplementary information: model training](https://developer.arm.com/solutions/machine-learning-on-arm/developer-material/how-to-guides/build-arm-cortex-m-voice-assistant-with-google-tensorflow-lite/supplementary-information-model-training) section in this guide.
+If you build and run your code now, your ML model knows the words “up” and “down”, but your application assumes that the words are “yes” and “no”. Let’s update the references and the user interface so that the appropriate words are printed.
 
-# Convert the model
+1. Navigate to the `tensorflow/lite/micro/examples/micro_speech/` directory and open the file `micro_features/micro_model_settings.cc`.
 
-Starting from the trained model to obtain a converted model that can run on the controller itself, we need to run a conversion script: the [TensorFlow Lite converter](https://www.tensorflow.org/lite/convert). This tool uses clever tricks to make our model as small and efficient as possible, and to convert it to a TensorFlow Lite FlatBuffer. To reduce the size of the model, we used a technique called [quantization](https://www.tensorflow.org/lite/performance/post_training_quantization). All weights and activations in the model get converted from 32-bit floating point format to an 8-bit and fixed-point format, as you can see in the following command:
+    You see the following category labels:
 
-![Convert the model to the TensorFlow Lite format code image](https://raw.githubusercontent.com/COTASPAR/K66F/master/images/convert_model.png)
-This conversion will not only reduce the size of the network, but will avoid floating point computations that are more computationally expensive.
+    ```
+    const char* kCategoryLabels[kCategoryCount] = {  
+    "silence",  
+    "unknown",  
+    "yes",  
+    "no",  
+    };
+    ```
 
-To save time, we will skip this step and instead download the [tiny_conv.tflite](https://developer.arm.com/-/media/Files/downloads/Machine%20learning%20how-to%20guides/tiny_conv.tflite?revision=495eb362-4325-49b8-b3ba-3141df0c9b95&la=en&hash=0F37BA2C5DE95A1561979CDD18973171767A47C3).
+    The code uses this array to map the output of the model to the correct value. Because we specified our wanted_words<!--is that underscore correct?--> as “up, down” in the training script, we should update this array to reflect these words in the same order.
```

Comment:
Yes, the underscore is correct but it should actually be all caps "WANTED_WORDS" as on the [python training script](https://github.com/tensorflow/tensorflow/blob/5d4a29eaf590b4a3068ef4d0b7bea9d4f7bd9369/tensorflow/lite/micro/examples/micro_speech/train/train_micro_speech_model.ipynb).

Reply:
```suggestion
    The code uses this array to map the output of the model to the correct value. Because we specified our WANTED_WORDS as “up, down” in the training script, we should update this array to reflect these words in the same order.
```